one_shot_scripts/update_deck_format.js

```diff
@@ -0,0 +1,332 @@
+const mongoose = require('mongoose');
+const Deck = require('../models/deck');
+const Draft = require('../models/draft');
+const CardRating = require('../models/cardrating');
+const mongosecrets = require('../../cubecobrasecrets/mongodb');
+var carddb = require('../serverjs/cards.js');
+var cubefn = require('../serverjs/cubefn.js');
+
+const batch_size = 100;
+
+function arrayIsSubset(needles, haystack) {
+  return needles.every((x) => haystack.includes(x));
+}
+
+function botRating(botColors, card, rating) {
+  const colors = card.colors || card.details.color_identity;
```

Comment:
This handles colorless overrides incorrectly doesn't it since it's an empty array.

Reply:
I think that's ok, we're just updating old decks anyway

Reply:
Bot decks are going to be far from ideal anyway